### PR TITLE
[Images] Updates to pricing page

### PR DESCRIPTION
**Pull request by crwaters16**

### CODE REVIEW

File: content/images/pricing.md [hunk 1]
```diff
@@ -6,15 +6,33 @@ weight: 6
...
\ No newline at end of file
+{{</Aside>}}
+
+For more information on how you can save costs by increasing caching in browsers, refer to [Browser TTL](/images/manage-images/browser-ttl/).
+
+## Images Light
+
+The Images Light plan lets you transform images that are stored outside of Cloudflare Images without purchasing any storage.
+
+Each unique transformation is billed once per 30 day period, regardless of cache. The 30-day period begins when the unique transformation is first made.
```
Comment (ranbel) [inline content/images/pricing.md]: Nit: Make the hyphenation consistent, i.e. "30 day period" vs "30-day period"

File: content/images/pricing.md [hunk 2]
```diff
@@ -6,15 +6,33 @@ weight: 6
...
+
+To use Images Light, select **Use my own storage** when purchasing Images in the Dash.
```
Comment (ranbel) [inline content/images/pricing.md]:
```suggestion
To use Images Light, select **Use my own storage** when purchasing Images in the dashboard.
```

**Review by ranbel — APPROVED** [review]

File: .github/actions/assign-issue/package-lock.json
Comment (pedrosousa) [inline .github/actions/assign-issue/package-lock.json:1]: I'd remove these two `package-lock.json` files from the PR.

File: content/images/pricing.md [hunk 3]
```diff
@@ -6,15 +6,33 @@ weight: 6
...
-|-----------------------------------|---------------|------------------|--------------------|
-| Free, Pro, Basic                  | $5.00 per 100K images stored per month | $1.00 per 100K images delivered per month | - $0.50 per 1K unique transformations per month<br> - 5K transformations included
-| Enterprise      | - $5.00 per 100K images stored per month<br> - 500K images included per month | - $1.00 per 100K images delivered per month<br> - 5M images included per month | $0.50 per 1K unique transformations per month<br> 1M transformations included |
+## Cloudflare Images
+
+Cloudflare Images lets you store, optimize, and deliver images from one unified API.
+
+|<div style="width:150px">Plan</div>| Images stored | Images delivered |
+|-----------------------------------|---------------|------------------|
+| Free, Pro, Basic                  | $5.00 per 100K images stored per month | $1.00 per 100K images delivered per month
```
Comment (crwaters16) [inline content/images/pricing.md]:
```suggestion
| Free, Pro, Business                  | $5.00 per 100K images stored per month | $1.00 per 100K images delivered per month
```

File: content/images/pricing.md (hunk 1 above)
Comment (crwaters16) [inline content/images/pricing.md]:
```suggestion
Each unique transformation is billed once per 30-day period, regardless of cache. The 30-day period begins when the unique transformation is first made.
```

**Pull request merged**